### DISCUSSION TITLE
Light UI theme

### DISCUSSION THREAD

**Discussion by bhalperin** [mockoon/mockoon#369] [Ideas] [answered]
Any plans to add light UI theme?

**Comment by 255kb** [comment, marked as answer]
There is an open issue (#100) and some WIP on a branch but highly outdated and not in any way finished. It will come for sure, but I cannot give any ETA considering the amount of work we have on more requested features (CLI, etc). 

You can see the roadmap there: https://github.com/orgs/mockoon/projects/2

**Reply by bhalperin** [reply]
Thanks for your reply. I'm very much enjoying using Mockoon!

**Reply by 255kb** [reply]
You're welcome, I am glad you like it :)
